rxbinding-design/src/main/java/com/jakewharton/rxbinding/support/design/widget/RxAppBarLayout.java

```diff
@@ -0,0 +1,22 @@
+package com.jakewharton.rxbinding.support.design.widget;
+
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.support.design.widget.AppBarLayout;
+import rx.Observable;
+
+/**
+ * Static factory methods for creating {@linkplain Observable observables} for {@link AppBarLayout}.
+ */
+public final class RxAppBarLayout {
+  /**
+   * Create an observable which emits offsetChange events for the {@code view}.
+   * <p>
+   * <em>Warning:</em> The created observable keeps a strong reference to {@code view}. Unsubscribe
+   * to free this reference.
+   */
+  @CheckResult @NonNull
+  public static Observable<AppBarLayoutOffsetChangeEvent> offsetChangeEvents(@NonNull AppBarLayout view) {
```

Comment:
We'll definitely want a non-event version of this that simply emits the offset integer. Avoid the overhead of creating all those objects.